### PR TITLE
Updated Lucene Indexers

### PR DESCRIPTION
**Pull request by enoriega**
The file structure of PMC OA changed on Dec 21st 2021. The xml files are split into different directories. This PR updates the NxmlIndexer class to handle this new structure.
Added a new class named PubmedAbstractIndexer to create an index of pubmed abstracts akin to the full-text OA index

### CODE REVIEW

**Comment by MihaiSurdeanu** [comment]
Nice, thank you @enoriega !

For future runs of this, can you please include a README in this directory to explain what steps need to happen to index: (a) all abstracts, and (b) all OA papers.
Thanks!

@kwalcock : if this looks Ok to you, can you please merge?

File: main/src/main/scala/org/clulab/reach/indexer/NxmlIndexer.scala [hunk 1]
```diff
@@ -171,7 +171,9 @@ object NxmlIndexer {
     val docsDir = props.getProperty("docs")
     val mapFile = props.getProperty("map")
 
+    println(props)
```
Comment (kwalcock) [inline main/src/main/scala/org/clulab/reach/indexer/NxmlIndexer.scala:174]: Just in case this is left over from debugging...

**Review by kwalcock — COMMENTED** [review]
Just one question (above).

File: main/src/main/scala/org/clulab/reach/indexer/NxmlIndexer.scala (hunk 1 above)
Comment (kwalcock) [reply main/src/main/scala/org/clulab/reach/indexer/NxmlIndexer.scala:174]: I'm assuming not.  It could be removed later if necessary.

**Comment by enoriega** [comment]
@kwalcock sorry about replying late. That print statement was already there and I found it useful when running it. Thanks for reviewing!

**Pull request merged**